cmd/apiserver/app/server/run_server.go

```diff
@@ -183,7 +183,7 @@ func (c checkEtcdConnectable) Check(_ *http.Request) error {
 	serverReachable, err := preflight.EtcdConnection{ServerList: c.ServerList}.CheckEtcdServers()
 
 	if err != nil {
-		glog.Error("etcd checker failed with err: %v", err)
+		glog.Errorf("etcd checker failed with err: %v", err)
```

Comment:
@ash2k better ask @MHBauer in a separate thread, I just fixed the formatting error.